src/main/resources/application.yml

```
@@ -13,6 +13,15 @@ spring:
       hibernate:
         dialect: org.hibernate.dialect.PostgreSQLDialect
     open-in-view: false
+
+# Auth0 Configuration
+auth0:
+  domain: ${AUTH0_DOMAIN:your-domain.auth0.com}
```

I think you can leave only environment variables here without default values. If you want to add an example of how these values should look like, do it via comment:
```suggestion
  domain: ${AUTH0_DOMAIN:} # your-domain.auth0.com
```